src/components/Table/Table.tsx

```diff
@@ -13,11 +13,24 @@ const Table = <TData,>({
   rowSize = 'sm',
   columnsConfig,
   sorting,
+  hasStickyHeader = false,
+  sx,
 }: TableProps<TData>) => {
   const { columnVisibility, setColumnVisibility } = columnsConfig ?? {};
 
   const hasColumnVisibilityConfig = Boolean(columnVisibility && setColumnVisibility);
 
+  /** If true, the scrollbar of tbody is visible */
+  const [hasScrollbar, setHasScrollbar] = React.useState(false);
+
+  const tBodyRef = useRef<HTMLTableSectionElement>();
+
+  React.useEffect(() => {
+    if (tBodyRef?.current) {
```

Comment:
do we need `useEffect` here? hasScrollBar could be true/false or null. If !null then it was set

Reply:
we need it in order for the hasScrollbar to be initialised after table is rendered 🤔

Reply:
but you can use if without `useEffect` right?

Reply:
if I dont put useEffect, I cannot get the values of scrollheight and clientHeight since tBodyRef is null on initial render. I use useEffect to trigger a re-render